test/integration/rules/landmark-unique/frame.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf8" />
+		<title>landmark-unique test</title>
+		<script src="/axe.js"></script>
+	</head>
+	<body>
+		<main id="violation-main-2">Second main</main>
```

Comment:
For this test can you add two landmarks, one with only a role and one with a non-empty name and a role. That way we can be certain that the name is actually being passed up the iframe chain.

Reply:
@waabid I should have been clearer, we also need an assertion on the actual values. I did not see one. If I missed, it, please point it out

Reply:
Sorry, I may not be following?

FTR; I added "violation-form-through-iframe-1", which should only fail if the role and name are being passed through iframes (and match the element in the parent element), as an example of when we are indirectly asserting for the values. For direct assertion, I think that might be another form of test (based off of what I've seen in integration tests): is that what you're asking for?